tck/build.gradle.kts

```diff
@@ -63,6 +62,7 @@ dependencyAnalysis {
         onAny {
             severity("fail")
             exclude("com.google.protobuf:protobuf-javalite")
+            onUnusedDependencies { exclude(":sdk") }
```

Comment:
Why is this needed?

Reply:
some check was failing before, let's see if it's fixed now

Reply:
the healthcheck fails, not sure why.

Reply:
It should not fail, but it does... Somehow the rename must confuse it. If you want you can keep it in and I will investigate later.

Reply:
alright